analytic_engine/src/instance/write.rs

```diff
@@ -486,6 +489,10 @@ impl Instance {
                           space.id,
                           self.db_write_buffer_size,
                     );
+
+                    let _timer = table_data
```

Comment:
Isn't this will drop timer above?

Reply:
It won't be dropped. But let's rename the outside timer to make it more explicit.